src/suews/src/suews_ctrl_driver.f95

```diff
@@ -3868,7 +3868,7 @@ SUBROUTINE SUEWS_cal_Main_DTS( &
             methodPrm%WaterUseMethod, irrPrm%Ie_start, irrPrm%Ie_end, &
             LAImethod, &
             evetrPrm%lai%laitype, dectrPrm%lai%laitype, grassPrm%lai%laitype, &
-            nsh_real, forcing%kdown, forcing%Temp_C, forcing%pres, BaseT_HC, &
+            nsh_real, forcing%kdown, forcing%Temp_C, forcing%rain, BaseT_HC, &
```

Comment:
Atmospheric pressure `pres` was mistakenly used in place of `rain` for the `dailystate` module.

Reply:
now fixed